### PR TITLE
update facet filer with multiple sections

### PR DESCRIPTION
**Pull request by skiran86**
update facet filer with multiple sections

### CODE REVIEW

File: src/components/SideBar/SideBarComponents/FacetFilters.js [hunk 1]
```diff
@@ -10,10 +11,12 @@ import {
   withStyles,
   Divider,
 } from '@material-ui/core';
+import _ from 'lodash';
 import CheckBoxIcon from '@material-ui/icons/CheckBox';
```
Comment (AjayDoddapaneni) [inline src/components/SideBar/SideBarComponents/FacetFilters.js:15]: import{  CheckBoxIcon, CheckBoxBlankIcon, ArrowDropDownIcon } from '@material-ui/icons';

**Review by AjayDoddapaneni — CHANGES_REQUESTED** [review]
minor change

File: src/utils/dashboardUtilFunctions.js [hunk 2]
```diff
@@ -403,6 +403,7 @@ export function customCheckBox(data) {
       checkboxItems: transformAPIDataIntoCheckBoxData(data[mapping.api], mapping.field),
       datafield: mapping.datafield,
       show: mapping.show,
+      section: mapping.section,
```
Comment (AjayDoddapaneni) [inline src/utils/dashboardUtilFunctions.js:406]: Can you check with Yizhen, weather this is updated in utils functions in his new upcoming PR? If not will break your changes.

Comment (skiran86) [reply src/utils/dashboardUtilFunctions.js:406]: Yep. Commented on PR # 244 and requested to save these changes.

**Comment by skiran86** [comment]
https://github.com/CBIIT/bento-tools/pull/17 is a dependency for this PR

File: src/utils/dashboardUtilFunctions.js [hunk 3]
```diff
@@ -0,0 +1,501 @@
+import uuid from 'uuid';
```
Comment (AjayDoddapaneni) [inline src/utils/dashboardUtilFunctions.js:1]: We don't need this file anymore

**Review by AjayDoddapaneni — CHANGES_REQUESTED** [review]
released 0.1.37 can you update it package json

**Pull request merged**